costs/anomalies/anomalies_route.go

```diff
@@ -67,6 +70,9 @@ func init() {
 func makeElasticSearchRequest(ctx context.Context, parsedParams AnomalyEsQueryParams, user users.User) (*elastic.SearchResult, int, error) {
 	l := jsonlog.LoggerFromContextOrDefault(ctx)
 	index := es.IndexNameForUser(user, "lineitems")
+	if len(parsedParams.IndexList) > 0 {
```

Comment:
Why do you do this like this and not like every other functions that you modified ?

Reply:
I forgot to remove it, good catch!